api/Clipboard.json

```diff
@@ -91,8 +91,19 @@
               "version_added": "79"
             },
             "firefox": [
+              {
+                "version_added": "90",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "dom.events.asyncClipboard.read",
+                    "value_to_set": "true"
+                  }
+                ]
```

Comment:
Sorry, @mirunacurtean, I missed something in our discussion earlier (since I didn't see the full context then). Can you copy the note ("Firefox only supports reading the clipboard…") from the next support statement here? That note applies to this support statement as well.

Based on https://bugzilla.mozilla.org/show_bug.cgi?id=1688865

Reply:
Done.